### PR TITLE
Fixes #501 by restoring marker-*pngs and bumping the npm/unpkg releas…

### LINKED ISSUE #501: Leaflet `marker-` assets not included in build.
I'm chasing down some 404 errors as I work with the new unpkg distribution.

Is there a reason that the `Gruntfile` explicitly grabs only the `layers` images?

```
src: ['layers.png','layers-2x.png'],
```

### PR DESCRIPTION
**Pull request by erictheise**
…e version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ defined in the MapML (Map Markup Language)
 
 Beginning with `v0.8.2`, &lt;mapml-viewer&gt; can be loaded from the [UNPKG content delivery network](https://unpkg.com/) –
 ```html
-<script type="module" src="https://unpkg.com/@maps4html/web-map-custom-element@0.8.2/dist/mapml-viewer.js"></script>
+<script type="module" src="https://unpkg.com/@maps4html/web-map-custom-element@0.8.3/dist/mapml-viewer.js"></script>
```
Comment (Malvoz) [inline README.md:15]: Should the README perhaps be version agnostic, and use `@latest` instead?

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ defined in the MapML (Map Markup Language)
 
 Beginning with `v0.8.2`, &lt;mapml-viewer&gt; can be loaded from the [UNPKG content delivery network](https://unpkg.com/) –
```
Comment (Malvoz) [inline README.md:13]: `v0.8.2` 🧐

**Review by Malvoz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (erictheise) [reply README.md:13]: Even though I unpublished that version for lacking the `marker-` pngs, it was the first working version available at UNPKG. I'll stand by that statement.

File: README.md (hunk 1 above)
Comment (erictheise) [reply README.md:15]: Absolutely.

File: README.md (hunk 2 above)
Comment (Malvoz) [reply README.md:13]: Ah, yes.

**Review by Malvoz — APPROVED** [review]

**Pull request merged**